webapp/client/src/pages/ConfirmationPage.js

```diff
@@ -0,0 +1,94 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { Trans, useTranslation } from "react-i18next";
+import FormFieldConsentBox from "../components/FormFieldConsentBox";
+import FormHeader from "../components/FormHeader";
+import StepForm from "../components/StepForm";
+import StepHeaderButtons from "../components/StepHeaderButtons";
+
+export default function ConfirmationPage({
+  stepHeader,
+  form,
+  fields,
+  dataPrivacyLink,
+  termsOfServiceLink,
+}) {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <StepHeaderButtons />
+      <FormHeader {...stepHeader} />
+      <StepForm {...form}>
+        <FormFieldConsentBox
+          required
+          fieldName="confirm_data_privacy"
+          fieldId="confirm_data_privacy"
+          checked={fields.confirmDataPrivacy.checked}
+          labelText={
+            <Trans
+              t={t}
+              i18nKey="unlockCodeRequest.fieldRegistrationConfirmDataPrivacy.labelText"
+              components={{
+                // The anchors get content in the translation file
+                // eslint-disable-next-line jsx-a11y/anchor-has-content
+                dataPrivacyLink: <a href={dataPrivacyLink} />,
+                taxGdprLink: (
+                  // eslint-disable-next-line jsx-a11y/anchor-has-content
+                  <a
+                    href="https://www.bundesfinanzministerium.de/Content/DE/Downloads/BMF_Schreiben/Weitere_Steuerthemen/Abgabenordnung/2020-07-01-Korrektur-Allgemeine-Informationen-Datenschutz-Grundverordnung-Steuerverwaltung-anlage-1.pdf?__blob=publicationFile&v=3"
+                    rel="noreferrer"
+                    target="_blank"
+                  />
+                ),
+              }}
+            />
+          }
+          errors={fields.confirmDataPrivacy.errors}
+        />
+        <FormFieldConsentBox
+          required
+          fieldName="confirm_terms_of_service"
+          fieldId="confirm_terms_of_service"
+          checked={fields.confirmTermsOfService.checked}
+          labelText={
+            <Trans
+              t={t}
+              i18nKey="unlockCodeRequest.fieldRegistrationConfirmTermsOfService.labelText"
+              components={{
+                // The anchors get content in the translation file
+                // eslint-disable-next-line jsx-a11y/anchor-has-content
+                termsOfServiceLink: <a href={termsOfServiceLink} />,
+              }}
+            />
+          }
+          errors={fields.confirmTermsOfService.errors}
+        />
+      </StepForm>
+    </>
+  );
+}
+
+const checkboxPropType = PropTypes.exact({
+  errors: PropTypes.arrayOf(PropTypes.string),
+  checked: PropTypes.bool,
+});
```

Comment:
Duplicates of this exist in several files now. That's bad because soon enough, someone will change one instance without changing the other and then we won't know if the divergence is meaningful or incidental.

Time to factor it out into a shared lib! I'd suggest something like `src/lib/propTypes.js`.